src/components/Table/Table.scss

```diff
@@ -52,18 +53,31 @@
   }
 }
 
+.ms-Table th,
+.ms-Table td,
 .ms-Table-cell {
   display: table-cell;
   padding: 0 10px;
 }
 
 // Style the first row as a header.
-.ms-Table-row:first-child {
+.ms-Table thead td,
```

Comment:
There can be, but it's best to use th in thead.  I'd be okay removing this line.